model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -145,7 +145,8 @@ object WorkspaceVersions {
 case class MultiCloudWorkspaceRequest(namespace: String,
                                       name: String,
                                       attributes: AttributeMap,
-                                      cloudPlatform: WorkspaceCloudPlatform
+                                      cloudPlatform: WorkspaceCloudPlatform,
+                                      region: String
```

Comment:
I'm still feeling somewhat uncomfortable with `region` being supplied in the request to create a workspace. I would imagine that most people would want the relay to be in the same region as their managed resource group, and therefore I think we should get the region somehow from azure based on the information in the azureConfig.

Should I start a conversation about this with regard to workspace landing zones? We will need to know pretty soon what "region" selection we are going to present in the "new workspace" dialog (though we could hardcode a region in the short term I guess).

In any case though, I feel like we could change where `region` comes from in a future ticket/PR, and for now we probably just want to move this PR through to unblock future work.

Reply:
Yeah, let's chat during today's tech talk, there some additional wrinkles w/WLZs that we need to consider. 

Agreed I think this is fine for now to block downstream UI work and integration.